.azure-pipelines-templates/install_deb.yml

```diff
@@ -35,11 +35,21 @@ steps:
       set -ex
       unsafe_build=$(grep unsafe /tmp/install_prefix) || true
       if [[ -n "$unsafe_build" ]]; then
-        ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON
+        ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON -DCOMPILE_TARGETS=sgx
       else
-        ./tests/test_install_build.sh
+        ./tests/test_install_build.sh -DCOMPILE_TARGETS=sgx
```

Comment:
While this proximally solves the issue, it is also the case that the current release .deb contains both sgx and virtual binaries, and so testing both of them is a feature! This is removing the test we have that the virtual artifacts we ship are usable, which is not what we want.

Reply:
@achamayou The step below builds the app in virtual, which tests the virtual artifacts.

Reply:
I'd somehow not seen those (or perhaps they've been added since?). Either way, this is probably fine for now, despite not being completely equivalent to the previous situation nor testing what happens for users who don't pass a COMPILE_TARGETS at all.